catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlQueryResponse.java

```diff
@@ -50,6 +51,8 @@
 
   private final Status status;
 
+  private QueryResponse queryResponse;
```

Comment:
✏️  If this is set in the ctor and never modified, we should be able to make it `final` as well.